lang/en.json

```diff
@@ -240,7 +276,7 @@
     "description": "Progress dialog text for partial flashes (quick)"
   },
   "flashing-full-flash-detail": {
-    "defaultMessage": "Initial flash might take longer, subsequent flashes will be quicker.",
+    "defaultMessage": "Your code is being sent ('flashed') to your micro:bit. This can take a while the first time, sorry, but it will be quicker after that.",
```

Comment:
I might look at temporarily reinstating the current French translation for this one.

Reply:
We can just defer pulling French back from Crowdin for now.